forge-gui/res/cardsfolder/upcoming/multiclass_baldric.txt

```diff
@@ -0,0 +1,13 @@
+Name:Multiclass Baldric
+ManaCost:1
+Types:Artifact Equipment
+K:Equip:2
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddKeyword$ Lifelink | IsPresent$ Cleric.YouCtrl | PresentCompare$ GE1 | Description$ Equipped creature has lifelink if you control a Cleric.
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddKeyword$ Deathtouch | IsPresent$ Rogue.YouCtrl | PresentCompare$ GE1 | Description$ Equipped creature has lifelink if you control a Rogue.
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddKeyword$ Haste | IsPresent$ Warrior.YouCtrl | PresentCompare$ GE1 | Description$ Equipped creature has lifelink if you control a Warrior.
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddKeyword$ Flying | IsPresent$ Wizard.YouCtrl | PresentCompare$ GE1 | Description$ Equipped creature has lifelink if you control a Wizard.
+R:Event$ DamageDone | ActiveZones$ Battlefield | Prevent$ True | ValidTarget$ Creature.EquippedBy | CheckSVar$ X | SVarCompare$ EQ4 | Description$ Prevent all damage that would be dealt to enchanted creature.
+SVar:X:Count$Party
+DeckHas:Ability$Party
+DeckHints:Type$Cleric|Rogue|Warrior|Wizard
+Oracle:Equipped creature has lifelink if you control a Cleric, deathtouch if you control a Rogue, haste if you control a Warrior, and flying if you control a Wizard.\nAs long as you have a full party, prevent all damage that would be dealt to equipped creature.\nEquip {2}
```

Comment:
All the descriptions need updating to fit with oracle

Reply:
Updated each SpellDescription$ to reflect Oracle formatting. Just to check for future reference, is the convention to include the entire Oracle paragraph in similar situations where multiple if-thens are contained to a single sentence? When looking at this card's card details pane in Forge, the "Equipped creature has ~ if ~" paragraph is copied four times. Is there a way to prevent this erroneous duplication?

Reply:
The description only on line 5 should be enough

Reply:
Updated, kept in only the description on line 5. The card detail pane now looks normal.